### PR TITLE
Remove unneeded checks if it runs on a Windows machine

### PR DESCRIPTION
**Pull request by MorrisJobke**
- the setup check is still there
- we don't support Windows

Try to review with whitespace ignored ;) Makes it easier.

cc @rullzer @LukasReschke @nickvergessen @schiessle

### CODE REVIEW

**Comment by rullzer** [comment]
The unit tests are not happy

File: tests/lib/Files/PathVerificationTest.php [hunk 1]
```diff
@@ -98,71 +98,14 @@ public function providesAstralPlane() {
...
-	/**
 	 * @dataProvider providesInvalidCharsPosix
 	 * @expectedException \OCP\Files\InvalidCharacterInPathException
 	 */
 	public function testPathVerificationInvalidCharsPosix($fileName) {
 		$storage = new Local(['datadir' => '']);
 
 		$fileName = " 123{$fileName}456 ";
-		self::invokePrivate($storage, 'verifyWindowsPath', [$fileName]);
+		self::invokePrivate($storage, 'verifyPosixPath', [$fileName]);
```
Comment (nickvergessen) [inline tests/lib/Files/PathVerificationTest.php]: woops :crying_cat_face:

**Comment by nickvergessen** [comment]
Fixed the one test, :+1: for not my part

**Comment by LukasReschke** [comment]
LGTM

**Pull request merged**